app/javascript/components/public_health/CustomExport.js

```diff
@@ -214,46 +214,90 @@ class CustomExport extends React.Component {
                     checked={this.state.selected_records === 'current'}
                   />
                   {this.state.selected_records === 'current' && (
-                    <div className="px-1 pb-1">
-                      <Badge variant="secondary" className="mr-1">
-                        {`${_.capitalize(this.props.patient_query.workflow)} - ${this.props.tabs[this.props.patient_query.tab]?.label}`}
-                      </Badge>
+                    <div className="custom-export-filters">
+                      <p className="custom-export-filter-text">
+                        <small>
+                          <b className="mr-2">Linelist</b>
```

Comment:
should we put a colon here to match the advanced filter ones? would also help with readability

Reply:
<img width="397" alt="Screen Shot 2021-07-14 at 11 28 46 AM" src="https://user-images.githubusercontent.com/35042815/125649200-f204caa3-9a95-4e3e-8292-9bd1cacc141c.png">

Reply:
same with jurisdiction, assigned user and search term

Reply:
done:

https://github.com/SaraAlert/SaraAlert/pull/980/commits/4cdc5386acf1f2d49a3076e5b48cda1354c3e0f6#diff-14793b3bc71143b45317a3d08474d370db9ffdfb9643e754341eeda1ab5f096bR220